src/components/JfDrawerComponent.spec.js

```diff
@@ -0,0 +1,28 @@
+import {shallowMount} from '@vue/test-utils';
+import JfDrawerComponent from './JfDrawerComponent.vue';
+
+describe('JfDrawerComponent', () => {
+    it('should set open variable to be true', () => {
+        const wrapper = shallowMount(JfDrawerComponent, {
```

Comment:
I think that it's better to wrap the shallowMount process into reusable fn, like
```
const shallowMountComponent = (Component, options, stubs)=>{
  return shallowMount(Component, {
    stubs,
    ...options
  }
};